pygdf/dataframe.py

```diff
@@ -353,7 +353,9 @@ def as_gpu_matrix(self, columns=None):
             raise ValueError('all column must have the same dtype')
         for k, c in self._cols.items():
             if c.has_null_mask:
-                raise ValueError("column {!r} is sparse".format(k))
+                errmsg = ("column {!r} has null values"
+                          "hint: use .fillna() to remove null values")
```

Comment:
should this say "hint: use .fillna() to replace null values"?  Saying "remove null values" makes me think of `.dropna()`